topics/computational-chemistry/tutorials/zauberkugel/tutorial.md

```diff
@@ -0,0 +1,311 @@
+>    - {% icon param-file %} *"Molecular input file"*: `output` (Input dataset)
+>    - *"Add hydrogens to polar atoms only (i.e. not to carbon atoms)"*: `Yes`
+>
+{: .hands_on}
+
+> ### {% icon question %} Question
+>
+> Nitrogen-containing functional groups are known to be basic. Which of them present in staurosporine do you expect to be protonated at pH 7.4, and which not? And why?
+>
+> > ### {% icon solution %} Solution
+> >
+> > Only the secondary N-methylamino group will be protonated because indoles much like aromatic amides are typically not basic.
+> >
+> {: .solution}
+>
+{: .question}
+
+## Splitting ePharmaLib into individual pharmacophores
+
+The ePharmaLib subset representing human protein targets (*ePharmaLib_PHARAO_human.phar*) is a concatenated file containing 5,010 individual pharmacophore files. In order to speed up our analysis, it is preferable to split the dataset into individual files in order to perform several pharmacophore alignments in parallel, using Galaxy's collection functionality.
+
+> ### {% icon hands_on %} Hands-on 5: Splitting ePharmaLib
+>
+> 1. {% tool [Split file](toolshed.g2.bx.psu.edu/repos/bgruening/split_file_to_collection/split_file_to_collection/0.5.0) %} with the following parameters:
+>    - *"Select the file type to split"*: `Generic`
+>        - {% icon param-file %} *"File to split"*: `output` (Input dataset)
+>        - *"Method to split files"*: `Specify record separator as regular expression`
+>            - *"Regex to match record separator"*: `\$\$\$\$`
+>            - *"Split records before or after the separator?"*: `After`
+>        - *"Specify number of output files or number of records per file?"*: `Number of records per file ('chunk mode')`
+>        - *"Base name for new files in collection"*: `epharmalib`
+>        - *"Method to allocate records to new files"*: `Maintain record order`
+>
+>    > ### {% icon details %} What is a PHAR file?
+>    >
+>    > A PHAR file is essentially a series of lines containing the three-dimensional coordinates of pharmacophoric features and excluded spheres. The first column specifies a feature type (e.g. HACC is a hydrogen bond acceptor). Subsequent columns specify the position of the feature center in a three-dimensional space. Individual pharmacophores are separated by lines containing four dollar signs (*$$$$*).
+>    {: .details}
+>
+{: .hands_on}
+
+## Ligand conformational flexibility
+
+In order to avoid combinatorial explosion, the Align-it ({% cite Taminau2008 %}) tool performs rigid alignment rather than flexible alignment. Conformational flexibility of the ligand is accounted for by introducing a preliminary step, in which a set of energy-minimized conformers for the query ligand are generated with the RDConf ({% cite rdconf %}) tool (using the RDKit ({% cite landrum2013rdkit %}) toolkit). 
```

Comment:
I understand you are making a point about the computational cost of flexible vs. rigid alignment, but I don't know what you mean by 'combinatorial explosion' specifically - can you elaborate or rephrase?

Reply:
I replaced 'In order to avoid combinatorial explosion' with 'To reduce the calculation time'.